js/app/widgets/eknWebview.js

```diff
@@ -176,9 +177,31 @@ const EknWebview = new Knowledge.Class({
                 if (scheme === 'license') {
                     let license = GLib.uri_unescape_string(uri.replace('license://', ''), null);
                     uri = Endless.get_license_file(license).get_uri();
+
+                    let license_view = new EosKnowledgePrivate.WebViewDialog({
+                      'transient-for' : this.get_toplevel(),
+                      'width-request' : 640,
+                      'height-request': 720,
```

Comment:
I tried default-width/height but its not respected, I guess the webview is requesting for more.

For example CC is responsive and it fits well in less than 640, but it does not fit so well in between  640 and 840

So I am not sure if there is an easy way to know which size the window should be.
Fo now I think the best think would be to open the view fullscreen!